### PR TITLE
Update Example Projects with Open Source Birthday

### PR DESCRIPTION
**Pull request by KarthikeyanRanasthala**

### CODE REVIEW

**Comment by KarthikeyanRanasthala** [comment]
I've resolved the merge conflict.

File: content/community/examples.md [hunk 1]
```diff
@@ -24,4 +24,5 @@ There are many example projects created by the React community. Feel free to add
 * **[Course Learn Page](https://github.com/ulearnpro/ulearn)** Open Source LMS script in Laravel 5.8 and ReactJS 16.9
 * **[Speedy math](https://github.com/pankajladhar/speedy-math)** An application which allows kids to practice basic Mathematics i.e  Addition, Subtraction, Multiply, Comparison. It is a PWA (Progressive web app) with offline support and install as App features.
 * **[Unit Converter](https://github.com/KarthikeyanRanasthala/react-unit-converter)** Minimal Yet Responsive Unit Converter Built With React, Material-UI & Convert-Units.
-* **[BMI Calculator](https://github.com/GermaVinsmoke/bmi-calculator)** A React Hooks app for calculating BMI
+* **[Open Source Birthday](https://github.com/KarthikeyanRanasthala/opensource-birthday)** Find your Open Source birthday by checking when you joined Github. Built with React, Semantic-UI, React-Redux, Redux-Thunk & Axios.
```
Comment (lex111) [inline content/community/examples.md]:
```suggestion
* **[Open Source Birthday](https://github.com/KarthikeyanRanasthala/opensource-birthday)** Find your Open Source birthday by checking when you joined GitHub. Built with React, Semantic-UI, React-Redux, Redux-Thunk & Axios.
```

**Comment by KarthikeyanRanasthala** [comment]
I've resolved the change request.

**Comment by gaearon** [comment]
Hi, we're trying to avoid Redux examples on this page.

**Pull request closed without merging**